types/crocks/crocks-tests.ts

```diff
@@ -0,0 +1,88 @@
+import crocks from 'crocks';
```

Comment:
```suggestion
import crocks = require('crocks');
```

Reply:
this will fix current CI error (when published, users can use ESM import as 'esModouleInterop' is usually true by default.

Reply:
I've commited this change, otherwise CI is failing. Once past this error, please review CI logs (if there is any other fail).
Otherwise looks good

Reply:
```suggestion
import crocks = require('crocks');
```